kolibri/deployment/default/settings/base.py

```diff
@@ -319,3 +319,7 @@
 JS_REVERSE_EXCLUDE_NAMESPACES = ['admin', ]
 
 ENABLE_DATA_BOOTSTRAPPING = True
+
+SESSION_EXPIRE_AT_BROWSER_CLOSE = True
+
+SESSION_COOKIE_AGE = 300
```

Comment:
https://docs.djangoproject.com/en/1.11/topics/http/sessions/#django.contrib.sessions.backends.base.SessionBase.get_expiry_age

5 minutes?

Reply:
Yes, 5 minutes. This is what we agreed with Harish.